docs/CNVs/CNV-methods.tex

```diff
@@ -670,125 +712,170 @@ \subsection{Generative Model for Read Counts} \label{coverage-model}
 %
 \begin{equation}
-n_{st} \sim {\rm Poisson}(d_s c_{st} e^{b_{st}})
+n_{st} \sim {\rm Poisson}(d_s \PP_{st} c_{st} e^{b_{st}})
 \label{coverage_model}
 \end{equation}
-%
-We can achieve many of the goals listed above by performing probabilistic PCA not on $\vn$ directly but rather on $\vb$.  This is one reason why we choose a Poisson parameter proportional to $e^b$ and not simply $b$ -- probabilistic PCA involves a normal distribution on $\vb_s$, whereas the Poisson parameter must be positive.  Furthermore, since PCA is a linear model with additive effects of different latent features, using $e^b$ represents multiplicative effects, which is more plausible.  Our model is
+
+We can achieve many of the goals listed above by performing probabilistic PCA not on directly $\vn$, but rather on $\vb$. One one hand, the Poisson parameters must be positive and therefore, $\exp(\vb)$ is a well-defined parametrization of the multiplicative bias. On the other hand, a Gaussian model for $\vb$ implies a log-normal distribution for $\exp(\vb)$ which is indeed the expected distribution when the multiplicative bias arises from several independent sources according to the central limit theorem\footnote{Let $B = \prod_{j=1}^{N_B} B_j$ be the total multiplicative bias where $B_j \in (0, \infty)$ are independent components of the bias. For $N_B \gg 1$, $\ln(B) \sim \mathcal{N}$ and therefore, $B$ has a log-normal distribution.}. We model $\vb$ as:
 %
 \begin{align}
-\vz \sim& {\rm N}(\vzero, \vI) \\
-\vb \sim& {\rm N}(\vW \vz + \vm, \vPsi),
+\vz &\sim \norm(\vzero, \vI), \nonumber\\
+\vb &\sim \norm(\vW \vz + \vm, \vPsi),
 \end{align}
 %
-where $\vz \in \mathbb{R}^D$ is a low-dimensional latent vector of laboratory conditions, $\vW \in \mathbb{R}^{T \times D}$ is a linear map from latent space to target space, $\vm \in \mathbb{R}^T$ is the vector of mean biases, and $\vPsi \in \mathbb{R}^{T \times T}$ is a diagonal matrix of residual variance not explained by the latent features.  We approximate the Poisson as a Gaussian centered at its mode (as a function of $b_{st}$):
+where $\vz \in \mathbb{R}^D$ is a low-dimensional latent vector of laboratory conditions, $\vW \in \mathbb{R}^{T \times D}$ is a linear map from latent space to target space, $\vm \in \mathbb{R}^T$ is the vector of mean biases, and $\vPsi \in \mathbb{R}^{T \times T}$ is a diagonal matrix of residual variance not explained by the latent features. We approximate the Poisson as a Gaussian (an excellent approximation for $n_{st} \gtrsim 1000$) and expand the argument of the Gaussian exponential about the mode of $b_{st}$ to quadratic order to obtain:
```

Comment:
The comment `an excellent approximation for $n_{st} \gtrsim 1000$` implies that the model only works for very high coverage.